prow/spyglass/lenses/metadata/lens.go

```diff
@@ -109,14 +109,12 @@ func (lens Lens) Body(artifacts []lenses.Artifact, resourceDir string, data stri
 			metadataViewData.Status = finished.Result
 		}
 	}
+	if metadataViewData.FinishedTime.IsZero() {
+		metadataViewData.FinishedTime = time.Now()
```

Comment:
Passing this into the template seems likely to at some point result in indicating that an ongoing job just finished (even if it doesn't right now), which I'm not really a fan of.

Reply:
yeahhh 🤷‍♂️